scripts/src/primazactl/cmd/create/namespace/common.py

```diff
@@ -81,7 +79,7 @@ def add_args_namespace(parser: argparse.ArgumentParser, type):
         help=f"Version of primaza to use, default: {__primaza_version__}. "
              "Ignored if --config is set.",
         type=semvertag_or_latest,
-        default=__primaza_version__)
+        default=None)
```

Comment:
Because command line options override options file we can't use defaults in arguments because they would always be specified. The defaults are still implemented though but in code rather than by argparse so help is still correct.